wbooks/src/app/screens/bookList/index.tsx

```diff
@@ -16,8 +16,15 @@ function BookList() {
 
   const itemSeparator = () => <View style={styles.separator} />;
 
+  const handleNavigateToHome = () => {
+    navigation.navigate('Home');
```

Comment:
Deberias tener un objeto que se llame Routes en donde ahi van a estar todos los nombres de las rutas de las pantallas, entonces aca lo que harias seria navigation.navigate(Routes.Home);

Reply:
a constants file is created containing the routes